container/docker/engine.py

```diff
@@ -317,6 +322,9 @@ def orchestrate(self, operation, temp_dir, hosts=[], context={}):
         command_options = self.DEFAULT_COMPOSE_UP_OPTIONS.copy()
         command_options[u'--no-build'] = True
         command_options[u'SERVICE'] = hosts
+        if 'detached' in locals():
```

Comment:
@j00bar Since `self.get_config_for_run()` calls `self.hosts_touched_by_playbook()` which calls `self.orchestrate('listhosts' ...)` which fails if `self.params['detached']` is True, so I have deleted `self.params['detached']` and set `detached=True`, [here](https://github.com/containscafeine/ansible-container/blob/0e2477b684831c83f8a0b478c4e4aba65987e178/container/docker/engine.py#L281)

I have used `detached=True` [here](https://github.com/containscafeine/ansible-container/blob/0e2477b684831c83f8a0b478c4e4aba65987e178/container/docker/engine.py#L325)